web/util.py

```diff
@@ -24,3 +25,13 @@ def flash_redirect(path: str, message: str | None = None, category="info"):
         flash(message, category)
 
     return redirect(path)
+
+
+def upload_file_to_printer(app, file, web_upload=False):
+    user_name = request.headers.get("User-Agent", "ankerctl").split(url_for('app_root'))[0]
+
+    with app.svc.borrow("filetransfer") as ft:
+        try:
+            ft.send_file(file, user_name)
+        except:
+            raise
```

Comment:
Okay, so:

```python3
try
    foo()
except:
    raise
```

is the same as

```python
foo()
```

:smile:


And at this point, the whole function is pretty small (still fine to keep, but maybe it's the `user_name` deduction we should make a helper for?

Reply:
✅